WordPress/Classes/ViewRelated/Jetpack/Install/View/JetpackRemoteInstallCardView.swift

```diff
@@ -0,0 +1,188 @@
+            imageView.widthAnchor.constraint(equalToConstant: Constants.iconWidth),
+            imageView.heightAnchor.constraint(equalToConstant: Constants.iconHeight)
+        ]
+    }
+
+    // MARK: Constants
+
+    struct Constants {
+        static let warningIconImage = UIImage(named: "jetpack-install-warning")
+        static let jetpackIconImage = UIImage(named: "jetpack-install-icon")
+        static let hideThisImage = UIImage(systemName: "eye.slash")
+        static let iconHeight: CGFloat = 30.0
+        static let iconWidth: CGFloat = 30.0
+        static let iconSpacing: CGFloat = -6.0
+        static let contentSpacing: CGFloat = 10.0
+        static let noticeLabelFont = WPStyleGuide.fontForTextStyle(.callout)
+        static let learnMoreFont = WPStyleGuide.fontForTextStyle(.callout).semibold()
+        static let contentDirectionalLayoutMargins = NSDirectionalEdgeInsets(top: -24.0, leading: 20.0, bottom: 12.0, trailing: 20.0)
+    }
+
+    struct Strings {
+        static let learnMore = NSLocalizedString("jetpackinstallcard.button.learn",
+                                                 value: "Learn more",
+                                                 comment: "Title for a call-to-action button on the Jetpack install card.")
+        static let hideThis = NSLocalizedString("jetpackinstallcard.menu.hide",
+                                                 value: "Hide this",
+                                                 comment: "Title for a menu action in the context menu on the Jetpack install card.")
+
+    }
+
+}
+
+// MARK: - JetpackRemoteInstallCardViewModel
+
+struct JetpackRemoteInstallCardViewModel {
+
+    let onLearnMoreTap: () -> Void
+    var noticeLabel: NSAttributedString {
+        switch installedPlugin {
+        case .multiple:
+            return NSAttributedString(string: Strings.multiplePlugins)
+        default:
+            let noticeText = String(format: Strings.individualPluginFormat, installedPlugin.rawValue)
+            let boldNoticeText = NSMutableAttributedString(string: noticeText)
+            guard let range = noticeText.nsRange(of: installedPlugin.rawValue) else {
+                return boldNoticeText
+            }
+            boldNoticeText.addAttributes([.font: WPStyleGuide.fontForTextStyle(.callout, fontWeight: .bold)], range: range)
+            return boldNoticeText
+        }
+    }
+
+    private let installedPlugin: JetpackPlugin
+
+    init(onLearnMoreTap: @escaping () -> Void = {},
+         installedPlugin: JetpackPlugin = .multiple) {
+        self.onLearnMoreTap = onLearnMoreTap
+        self.installedPlugin = installedPlugin
+    }
+
+    enum JetpackPlugin: String {
+        case search     = "Jetpack Search"
+        case backup     = "Jetpack Backup"
+        case protect    = "Jetpack Protect"
+        case videoPress = "Jetpack VideoPress"
+        case social     = "Jetpack Social"
+        case boost      = "Jetpack Boost"
+        case multiple
+    }
```

Comment:
@dvdchr I think there's more than this since the example was **Jetpack VaultPress Backup** here: pctCYC-An-p2#comment-728.

I based these off of the spreadsheet you created `Blog data returned from /me/sites`. Do we have a list of these somewhere or is that spreadsheet all of them?

Reply:
@wargcm 👋🏼. I manually populated that sheet by looking at the Jetpack plugins on the 'Install Plugins' page.  Another option is to look at the 'My Jetpack' page (at `/wp-admin/admin.php?page=my-jetpack`):

<img width="1120" alt="Screenshot 2023-02-15 at 11 03 25" src="https://user-images.githubusercontent.com/1299411/218926388-264b3afa-e363-4ea5-bd41-0bf235215f41.png">

But there are some oddities here:
1. 'Extras' is actually the full Jetpack plugin.
2. When you only have the full plugin installed, 'Protect' there appears as 'Scan'.

Some notes about **Akismet** and **Jetpack CRM**: they are also individual Jetpack plugins, but they don't show up in the `jetpack_connection_active_plugins`. Also, I've tested that you can't create a Jetpack connection just by having them installed, so it's not possible to reach/reproduce the broken state in the app just by having one of these.

> ... the example was Jetpack VaultPress Backup

Yes, that's the public-facing name for `jetpack-backup`. We've just been calling it Jetpack Backup internally, probably for short 😅 . For reference, I've mapped the values of `jetpack_connection_active_plugins` to its public name and added them in Appendix 2 here: pctCYC-An-p2#appendix

Reply:
👍 I updated the backup display name to `Jetpack VaultPress Backup` here: https://github.com/wordpress-mobile/WordPress-iOS/pull/20136/commits/4b960dd04cf0da3d58757969de60b6053969ca6d